onnxruntime/core/profile/profile.h

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <cinttypes>
+#include <cstdlib>
+#include <iostream>
+#include <stdexcept>
+#include <string>
+
+namespace onnxruntime {
+namespace profile {
+
+enum EventColor : uint32_t {
```

Comment:
>enum EventColor [](start = 0, length = 15)

consider making this an enum class and using EventColor in the interface instead of uint32_t

Reply:
Sure!

---
In reply to: [428990910](https://github.com/microsoft/onnxruntime/pull/4013#discussion_r428990910) [](ancestors = 428990910)